c/datasetjson.c

```diff
@@ -367,21 +367,20 @@ static int obtainDSCB1(const char *dsname, unsigned int dsnameLength,
       } parmList;
     )
   );
-
```

Comment:
There's a lot of random white space. This should be avoided if possible. I'm not super picky about it, but others are.